api/issue.go

```diff
@@ -102,7 +102,25 @@ type IssueCreate struct {
 	AssigneeID       int       `jsonapi:"attr,assigneeId"`
 	SubscriberIDList []int     `jsonapi:"attr,subscriberIdList"`
 	RollbackIssueID  *int      `jsonapi:"attr,rollbackIssueId"`
-	Payload          string    `jsonapi:"attr,payload"`
+	// CreateContext is used to create the issue pipeline and not persisted.
+	CreateContext string `jsonapi:"attr,createContext"`
```

Comment:
nit:  I would put CreateContext after payload since it's a special field and not persisted, which is different from other fields.

Reply:
Also if it's just for create database, I would name it CreateDatabaseContext.